rskj-core/src/test/java/co/rsk/net/SnapshotProcessorTest.java

```diff
@@ -0,0 +1,206 @@
+package co.rsk.net;
```

Comment:
Missing copyright header. I also forgot it [in TrieDTOInOrderIteratorTest](https://github.com/rsksmart/rskj/pull/2233/files#diff-362bf857c47e29336ba5ea4647c9b16aaec30112019a6836765f04bb58efbed6), I would appreciate if you could add it there aswell

Reply:
Done